templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,13 +1,16 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.3</feature>
+        <feature>servlet-4.0</feature>
```

Comment:
I don't think we need the servlet feature... we do want to explicitly add cdi-2.0 even if unnecessary.